### PR TITLE
Use IconCollection

### PR DESCRIPTION
**Pull request by alainm23**
Separating the list of icons in a new class.

### CODE REVIEW

**Review by danirabbit — CHANGES_REQUESTED** [review]
Don't forget to add new files to POTFILES, otherwise looks good!

**Comment by alainm23** [comment]
I always forget 😅️

File: po/POTFILES [hunk 1]
```diff
@@ -1,5 +1,6 @@
 src/Application.vala
 src/CategoryView.vala
+src/CategoryView.vala
```
Comment (danirabbit) [inline po/POTFILES:3]: Should be IconCollection no?

**Review by danirabbit — APPROVED** [review]
Looks good! Thanks :)

**Pull request merged**